network/p2p/acp118/handler_test.go

```diff
@@ -125,6 +125,13 @@ func TestHandler(t *testing.T) {
 				require.NoError(err)
 
 				require.True(bls.Verify(pk, signature, request.Message))
+
+				if tt.cacher.Len() > 0 {
+					// Ensure the cache is populated with correct signature
+					sig, ok := tt.cacher.Get(unsignedMessage.ID())
+					require.True(ok)
+					require.Equal(sig, response.Signature)
+				}
```

Comment:
Rather than checking the length (which seems fairly fragile) we should just assert that the signature is as expected if the cacher returns the signature.

Reply:
done